src/annotator/sidebar.ts

```diff
@@ -76,60 +82,86 @@ function createSidebarIframe(config) {
   return sidebarFrame;
 }
 
+type GestureState = {
+  /** Initial position at the start of a drag/pan resize event (in pixels). */
+  initial: number | null;
+  /** Final position at end of drag resize event. */
+  final: number | null;
+};
+
 /**
  * The `Sidebar` class creates (1) the sidebar application iframe, (2) its container,
  * as well as (3) the adjacent controls.
- *
- * @implements {Destroyable}
  */
-export class Sidebar {
+export class Sidebar implements Destroyable {
+  private _emitter: Emitter;
+  private _config: SidebarContainerConfig & SidebarConfig;
+  private _listeners: ListenerCollection;
+  private _gestureState: GestureState;
+  private _layoutState: SidebarLayout;
+  private _hammerManager: any | undefined; // FIXME Exported types from hammerjs are all messed-up
```

Comment:
Checking this at the moment, but I'm struggling to find the proper types so that Typescript does not complain.